src/main/java/collision/CollisionManager.java

```diff
@@ -0,0 +1,55 @@
+package collision;
+
+import object.GameObject;
+import object.GameObjectManager;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+public class CollisionManager {
+
+    private Collection<Collision> collisions;
+    private GameObjectManager gameObjectManager;
+
+    public CollisionManager(GameObjectManager gameObjectManager) {
+        this.collisions = new HashSet<>();
+        this.gameObjectManager = gameObjectManager;
+    }
+
+    public void processCollisions() {
```

Comment:
lets pass collision collection here, using argument

Reply:
who would create this collection, and why do that?